### PR TITLE
Added my LinkFree profile

### PR DESCRIPTION
**Pull request by nanthakumaran-s**
## Changes proposed
Added `Nanthakumaran-S.json` under `data/`

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2636"></a>

### CODE REVIEW

**Review by ChinmayMhatre — APPROVED** [review]
LGTM!

File: data/Nanthakumaran-S.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "name": "Nanthakumaran S",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Software Developer | **Cloud** & **DevOps** Enthusiast | Open Source Beginner",
+  "tags": ["Open Source", "DevOps", "Kubernetes", "Docker", "Node.js", "React.js"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/nanthakumaran_" },
+    { "icon": "FaGithub", "url": "https://github.com/nanthakumaran-s" },
+    { "icon": "FaLinkedin", "url": "https://linkedin.com/in/nanthakumaran-s" },
+    { "icon": "website", "url": "https://nanthakumaran.com" }
```
Comment (eddiejaoude) [inline data/Nanthakumaran-S.json:11]: This is not a valid icon, please select from `Fa` or `Si` icon set, you can search the here https://linkfree.eddiehub.io/icons

Btw I see many people using `website` is this somewhere in the docs that we forgot to update?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
The filename does not match the username case.

File: data/Nanthakumaran-S.json (hunk 1 above)
Comment (nanthakumaran-s) [reply data/Nanthakumaran-S.json:11]: Sure will change them... I have found similar usage in others' JSON files.

**Comment by nanthakumaran-s** [comment]
> The filename does not match the username case.

Could you please help me to understand this... I have given my file the name of `Nanthakumaran-S.json` and inside that, my name is written as `Nanthakumaran S`. What are the things I need to change in the following case?

**Comment by eddiejaoude** [comment]
Sure, your username on GitHub is lower case, the file needs to be your exact username from GitHub 

Screenshot 2023-01-04 at 08 10 21

**Comment by nanthakumaran-s** [comment]
Fixed the issue. Thanks for helping me out

**Review by eddiejaoude — APPROVED** [review]
Thank you for fixing, looks good 👍

**Pull request merged**